src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -71,6 +73,13 @@ public void setPerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
     }
 
     /**
+     * Sort list by name
+     */
+    public void sort() {
+        Collections.sort(internalList, Comparator.comparing(firstPerson -> firstPerson.getName().fullName));
```

Comment:
Small thing: notice that the sort-function is case-sensitive, meaning that if a person's name is not starting with a capitalized letter, it will end up in the end of the list after sorting if all the other names start with a capitalized letter.